### ISSUE TITLE
Allow color customization of text in room schedules

### ISSUE THREAD

**Issue by Samuel-Baird** [thewca/worldcubeassociation.org#7549]
![image](https://user-images.githubusercontent.com/45566917/213606786-22c4460f-3779-4bd1-bec3-513a69351610.png)
Currently if a white background is used for a room on a competition website schedule then it becomes very difficult to read the text. It would be great to have some text customization options (even if the options are just white and black)

**Comment by dunkOnIT** [comment]
Seems reasonable, although I do wonder if another solution might just be disabling white as a room colour? I can imagine a world where other colour combinations are difficult to see, though, so text colour customisation would still seem like a good way forward.

**Comment by Samuel-Baird** [comment]
> Seems reasonable, although I do wonder if another solution might just be disabling white as a room colour? I can imagine a world where other colour combinations are difficult to see, though, so text colour customisation would still seem like a good way forward.

I’m not sure how difficult it would be for WST to block off a part of the color spectrum, the color used in the photo isn’t even white, but rather an off-color white to make the text a bit easier to see. 

In this case we are using white and red because those are the colors of tablecloths that the venue has available for us for our 2 stages. It just makes an easier to have stage color match with schedule color :p

**Comment by JonEsparaz** [comment]
Related to (maybe duplicate of) #7276.
